processing/src/main/java/io/druid/query/groupby/GroupByQueryConfig.java

```diff
@@ -35,6 +35,7 @@
   private static final String CTX_KEY_BUFFER_GROUPER_MAX_SIZE = "bufferGrouperMaxSize";
   private static final String CTX_KEY_MAX_ON_DISK_STORAGE = "maxOnDiskStorage";
   private static final String CTX_KEY_MAX_MERGING_DICTIONARY_SIZE = "maxMergingDictionarySize";
+  public static final String CTX_KEY_SKIP_NULL_DIMENSION = "groupBySkipNullDimension";
```

Comment:
suggest changing the key name to indicate null "dimension values" instead of just "dimension"; using "null dimension" sounds like this option only applies to dimensions where all values are null